src/data/HarnessAssembly.js

```diff
@@ -1,7 +1,27 @@
+import WireHarness1 from "@/public/images/wireharness/WireHarness1.webp";
```

Comment:
You edited a different file. Check the issue, we would like you to create a new file